docs/source/advanced_usage.rst

```diff
@@ -377,11 +346,10 @@ The following shows an instructive example of all three.
     RZ(-2.0) 0
     H 0
 
-A more sophisticated feature of pyQuil is that it can create templates of Quil programs in
-ParametricProgram objects.  An example use of these templates is in exponentiating a Hamiltonian
-that is parametrized by a constant.  This commonly occurs in variational algorithms. The function
-``exponential_map`` is used to compute exp[-i * alpha * H] without explicitly filling in a value for
-alpha.
+The ``exponential_map`` function returns a function allowing you to fill in a multiplicative
```

Comment:
Maybe change
> The `exponential_map` function

to just
> `exponential_map`

since it sounds weird to have the word "function" appear twice in a sentence.

Reply:
changed